tsdb/head.go

```diff
@@ -1010,6 +1010,13 @@ func (h *RangeHead) Meta() BlockMeta {
 	}
 }
 
+// String returns an human readable representation of the range head. It's important to
+// keep this function in order to avoid the struct dump when the head is stringified in
+// errors or logs.
+func (h *RangeHead) String() string {
+	return "range head"
```

Comment:
We could include the mint and maxt for this range head here

Reply:
Sure. What do you think?